tests/scheduler/test_slurm.py

```
@@ -72,6 +74,40 @@ def test_get_runtime_file_name():
+
+    # Add file handler to help
+    project.logger.addHandler(logging.FileHandler("test.log"))
+
+    class DummyPOpen:
+        def wait(self):
+            return
+
+        returncode = 10
+
+    # Run the project's build process synchronously.
+    with patch("shutil.which") as which, patch("subprocess.Popen") as popen, patch("siliconcompiler.schema_support.record.RecordSchema.record_python_packages") as record_python_packages:
```

Ruff (0.14.4)

Local variable `record_python_packages` is assigned to but never used

Remove assignment to unused variable `record_python_packages`

(F841)